plugins/amt/fu-plugin-amt.c

```diff
@@ -416,7 +416,8 @@ fu_plugin_amt_create_device (GError **error)
 	memcpy (&ver, response->data, sizeof(struct amt_code_versions));
 
 	dev = fu_device_new ();
-	fu_device_set_id (dev, "/dev/mei");
+	fu_device_set_id (dev, "/dev/mei0");
+	fu_device_set_platform_id (dev, "/dev");
```

Comment:
I think you just want one call to `fu_device_set_platform_id (dev, "/dev/mei0");` -- if you don't have any GUIDs set then the value set as the platform ID is also just used as the ID.

Reply:
fixed